library/src/main/java/com/otaliastudios/zoom/ZoomEngine.kt

```diff
@@ -1235,34 +1356,81 @@ internal constructor(context: Context) : ViewTreeObserver.OnGlobalLayoutListener
         dispatchOnMatrix()
     }
 
+    /**
+     * Converts an [AbsolutePoint] to a [ScaledPoint]
+     */
+    private fun AbsolutePoint.toScaled(): ScaledPoint {
```

Comment:
Nice. I think we could finally remove `resolvePan` and `unresolvePan` and create extensions of the `Float` class with the same names?

Reply:
I mean to call them toScaled() and toAbsolute() of course.

Reply:
Sure.